src/TypesGenerator.php

```diff
@@ -635,6 +635,8 @@ private function generateField(array $config, array $class, \EasyRdf_Resource $t
 
             if (isset($propertyConfig['nullable']) && false === $propertyConfig['nullable']) {
```

Comment:
What about the following?

```suggestion
            if (isset($propertyConfig['nullable'])) {
                $isNullable = (bool) $propertyConfig['nullable'];
            }
```